packages/blockchain-wallet-v4-frontend/src/middleware/analyticsMiddleware/index.ts

```diff
@@ -305,13 +330,18 @@ const analyticsMiddleware = () => (store) => (next) => (action) => {
             const inputAmount = Number(state.form.swapAmount.values.cryptoAmount)
             const inputCurrency = state.form.initSwap.values.BASE.coin
             const inputType =
-              state.form.initSwap.values.BASE.type === 'CUSTODIAL' ? 'TRADING' : 'USERKEY'
+              state.form.initSwap.values.BASE.type === 'CUSTODIAL'
```

Comment:
There is a branch that's not merged in yet where I created a `SwapBaseCounterTypes` enum for this. You may want to add a TODO or make a note to come back and change this later.